CONTRIBUTING.md

```diff
@@ -339,6 +339,9 @@ harmless.  This is a strictly better compromise than using the latter
 two forms, which would generate false positive errors for correct use
 under Python 3.7.4.
 
+When your stub contains different code for different Python versions,
+always put the most recent Python version first.
```

Comment:
```suggestion
When your stub contains if statements for different Python versions,
always put the code for the most recent Python version first.
```
nit: minor rephrase since we only do this on a per if statement level, e.g. this is okay
```
if sys.version:
    new stuff
else:
    old stuff that comes before later new stuff
...
if sys.version:
    later new stuff
```